ra/registration-authority.go

```diff
@@ -43,15 +45,21 @@ type RegistrationAuthorityImpl struct {
 	log         *blog.AuditLogger
 	// How long before a newly created authorization expires.
 	authorizationLifetime time.Duration
+	rlPolicies            cmd.RateLimitConfig
+	tiMu                  *sync.RWMutex
+	totalIssuedCache      *int64
```

Comment:
this doesn't need to be a pointer since RA's always take a `*RegistrationAuthorityImpl`, no?